runtime/near-vm-runner-standalone/src/main.rs

```diff
@@ -3,7 +3,7 @@
 mod script;
 
 use crate::script::Script;
-use clap::Clap;
+use clap::Parser;
```

Comment:
Is this use necessary at all, since the derive is using `clap::Parser` in the first place

Reply:
@nagisa That's a good question. 
We are importing `clap::Parser` which is used as trait. It's needed for `let cli_args = CliArgs::parse();`.